test.js

```diff
@@ -14,34 +14,34 @@ const type = 'log'
 const consistency = 'one'
 const host = 'localhost'
 const port = 9200
+// new data in Elasticsearch is only visible after the refreshInterval
+const refreshInterval = 1100 // assume ES default settings 1 second
```

Comment:
The test could force a refresh after indexing. One more query to run before results can be checked: 
https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-refresh.html